File-Formats/PDF-to-image/Convert-PDF-file-to-Image-in-WPF.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Convert PDF file to Image in WPF application | Syncfusion
-description: Learn how to convert a PDF file to Image in WPF with easy steps using System Drawing library.
+title: Convert PDF file to Image in WPF application | Syncfusion&reg;
```

Comment:
Since we have mentioned as "document" in description change "file" to "document"

Reply:
I have modified